ksql-engine/src/test/resources/query-validation-tests/window-times.json

```diff
@@ -0,0 +1,159 @@
+{
+  "comments": [
+    "Test cases covering WindowStartTime UDAF"
```

Comment:
We should add a case to validate that the result can be used in expressions (e.g. something like ```SELECT ... WindowStart() / 1000 AS START_SECONDS ... ```)

Reply:
I'll add something like that - however, I would imagine at the moment this won't work - so just need to make sure it fails in a meaningful way.